TINY-10172: Prevent scroll when showing the Throbber (#9042)
tinymce/tinymce: modules/sugar/CHANGELOG.md

## Unreleased

### Improved

> - The `Focus.focus` function now takes an additional parameter of type boolean that when passed with the value `true` then the element would be focused without being scrolled.

Reviewer: ```suggestion
- The `Focus.focus` function now takes an additional `preventScroll` parameter to allow focus on an element without scrolling.
```
